### DISCUSSION TITLE
Dragging window to the sides changes space/desktop

### DISCUSSION THREAD

**Discussion by tsambou** [rxhanson/Rectangle#1328] [General]
Not exactly a bug but a difficulty 

macOS version: ventura 13.4

My macbook is automatically changing spaces when dragging an app to the sides of the screen. I tried looking for a way to disable this feature in mac (which btw hadn't noticed before it existed) but didn't find something. Because I wanted to use this app only for the manual use (no shortcuts) it renders it unusable for me at the moment.

Is there a workaround for this?

Thanks

**Comment by rxhanson** [comment]
Thanks for the feedback. For most people that have this issue, they increase the margin for the snap areas via terminal command:

https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#changing-the-margin-for-the-snap-areas

**Reply by chuim** [reply]
It would be interesting if there it was possible to configure these margins in the preferences UI. Would it be OK to file an enhancement issue for this request? I think that a single snap size control that affects all 4 snap margin directions would be good enough.
